### ISSUE TITLE
Remove center images AstroDrizzle outputs

### ISSUE THREAD

**Issue by ktfhale** [STScI-Citizen-Science/MTPipeline#145]
Now that the alpha CosmoQuest-preserving release has been made, and once the revise_PNG branch is merged into master, (Ticket #144), I'll create a new branch `remove-center`. In this branch I'll get rid of the logic in `run_astrodrizzle` that produces the center images. I'll also remove the center cfg files from the repository.

This will be the first step in changing our pipeline outputs to comply with MAST (Ticket #143).

**Comment by ktfhale** [comment]
I've run this on `_flt.fits` and `_c0m.fits` inputs, and... everything seems to work. I kept in the for-loop loop over all the configuration files for each detector, even though now there's only the wide cfg file. That is, I thought I'd keep the easy ability to add more AstroDrizzle "modes". I think this is ready to be merged.

I've also removed all the center files from the archive.
